service_mesh/v2x/ossm-traffic-manage.adoc

```diff
@@ -42,6 +42,11 @@ include::modules/ossm-auto-route.adoc[leveloffset=+2]
 
 include::modules/ossm-auto-route-annotations.adoc[leveloffset=+2]
 
+ifdef::openshift-enterprise[]
+.Additional resources
+* xref:../../networking/routes/route-configuration.adoc[Special annotations].
```

Comment:
Did you want to link to the Assembly or to the route-specific annotations topic in this assembly?

https://docs.openshift.com/container-platform/4.11/networking/routes/route-configuration.html#nw-route-specific-annotations_route-configuration 

A tip for creating the XREF is to copy the URL for the specific heading you want, and then replace the first part of the URL with the ../../ for the relative path, and the HTML with adoc.  So the XREF for the heading above would be:

`xref:../../networking/routes/route-configuration.adoc#nw-route-specific-annotations_route-configuration[HEADING TEXT]`

Reply:
Thanks for that tip @JStickler! The PR and preview build have been updated. If this looks good, we can merge it.